homeassistant/components/decora_wifi/const.py

```diff
@@ -0,0 +1,11 @@
+"""Constants for decora_wifi."""
+DOMAIN = "decora_wifi"
+
+CONF_FAN = "fan"
+CONF_LIGHT = "light"
```

Comment:
Correct me if I'm wrong, but I would have thought it would be incorrect to re-use DOMAIN from LightEntity since not every entity supported by this integration will be a LightEntity. In addition to smart lights/dimmers, python-decora_wifi also supports (among other things) fan speed controllers and smart switches that could control things other than lights.

Reply:
Not reuse, but import under a different name, `from ... Import DOMAIN as LIGHT_DOMAIN`

Reply:
Ohh, and use that instead of making up my own CONF_LIGHT, etc.  That makes sense.